src/lite-yt-embed.js

```diff
@@ -98,13 +110,31 @@ class LiteYTEmbed extends HTMLElement {
         LiteYTEmbed.preconnected = true;
     }
 
-    addIframe(){
-        const iframeHTML = `
-<iframe width="560" height="315" frameborder="0"
-  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen
-  src="https://www.youtube-nocookie.com/embed/${this.videoId}?autoplay=1"
-></iframe>`;
-        this.insertAdjacentHTML('beforeend', iframeHTML);
+    onYouTubeIframeAPIReady() {
+        const videoWrapper = document.createElement('div')
+        videoWrapper.id = this.videoId;
+        document.body.appendChild(videoWrapper);
+        this.insertAdjacentElement('beforeend', videoWrapper);
+
+        new YT.Player(videoWrapper, {
+            width: '100%',
+            videoId: this.videoId,
+            playerVars: { 'autoplay': 1, 'playsinline': 1 },
+            events: {
+                'onReady': (event) => {
+                    event.target.mute();
```

Comment:
Why mute?

Reply:
Removed. It was used for testing.
Thanks!